src/pages/features/index.md

```diff
@@ -51,6 +54,9 @@ The following are known limitations for the Alpha release
 
 [Here](../code-sample/#example-17-edit-text-layers) is a code example.
 
+Example of Text with a sample image. Here font is changed from the original image.
```

Comment:
"Here is an example where the font was changed from the original image on the left using the Text API."